### ISSUE TITLE
3.11.0 : NullPointerException on AdView.getAdSize()

### ISSUE THREAD

**Issue by Myke38** [googleads/googleads-mobile-unity#609]
Hello,

I am trying to get the pixel size of a BannerView with the function GetWidthInPixels() on Android,
but I have a NullPointerException error in my logcat  :

```
Unity   : AndroidJavaException: java.lang.NullPointerException: Attempt to invoke virtual method 'com.google.android.gms.ads.AdSize com.google.android.gms.ads.AdView.getAdSize()' on a null object reference
Unity   : java.lang.NullPointerException: Attempt to invoke virtual method 'com.google.android.gms.ads.AdSize com.google.android.gms.ads.AdView.getAdSize()' on a null object reference
Unity   :      at com.google.unity.ads.Banner.getWidthInPixels(Banner.java:320)
Unity   :      at com.unity3d.player.UnityPlayer.nativeRender(Native Method)
Unity   :      at com.unity3d.player.UnityPlayer.c(Unknown Source)
Unity   :      at com.unity3d.player.UnityPlayer$c$1.handleMessage(Unknown Source)
Unity   :      at android.os.Handler.dispatchMessage(Handler.java:98)
Unity   :      at android.os.Looper.loop(Looper.java:154)
Unity   :      at com.unity3d.player.UnityPlayer$c.run(Unknown Source)
Unity   :   at UnityEngine.AndroidJNISafe.CheckException () [0x00000] in <filename unknown>:0
Unity   :   at UnityEngine.AndroidJNISafe.CallFloatMethod (IntPtr obj, IntPtr methodID, UnityEngine.jvalue[] args) [0x00000] in <filename unknown>:0
Unity   :   at UnityEngine.And
```

Please can you tell me what is the problem ?
Thank you
Michael

**Comment by rampara** [comment]
@Myke38 can your provide the code snippet you used to reproduce this exception?

**Comment by Myke38** [comment]
Hello, yes sure, here is the code : 
```
MobileAds.Initialize(appId);
BannerView bannerView;
bannerView = new BannerView(adBannerId, AdSize.MediumRectangle,AdPosition.TopLeft);
bannerView.LoadAd(CreateGenericRequest());	
float widthPixelAd = bannerView.GetWidthInPixels(); // <--------- EXCEPTION HERE --------
```
And the GenericRequest function : 
```
private AdRequest  CreateGenericRequest(){
	return new AdRequest.Builder()
		.AddTestDevice(deviceTest1).AddTestDevice(deviceTest2)
	        .AddKeyword("game").SetGender(Gender.Male)
		.Build();
}
```

The purpose of getting the size of the banner is to set the position of the banner at a specific location, with the function :
`
bannerView.SetPosition(...);`

Thank you for your help
Michael

**Comment by rampara** [comment]
@Myke38 the steps you outlined works as expected for me. Can you provide a Unity test app that repros your problem?

**Comment by Myke38** [comment]
Hello, 
Here is the Unity test app :
https://we.tl/NO18Z6yfu3

To reproduce the bug :
- Open the project with Unity 5 (5.6.4 for me)
- Build for Android platform, and send the APK on a device
- Click one time on the button : the banner is well positionning, with no error.
- Click a second time on the button to get the exception.

Please tell me if you can reproduce the same issue.
Thank you

**Comment by rampara** [comment]
@Myke38 Thanks for providing the test app. There will be a fix for this issue pushed out shortly.

**Comment by rampara** [comment]
This issue is fixed in the latest version of the plugin (v3.11.1).
